### PR TITLE
Fix display problem when moving the app window to secondary monitor (V 0.1.27 - W10)

### PR DESCRIPTION
**Pull request by progamateur**
& fix some UI V 0.0.3 compatibilities
v fmt not used as it  gives bad files formatting results.

### CODE REVIEW

**Comment by medvednikov** [comment]
macos:
```
`/Users/runner/runners/2.262.1/work/ui/ui/vlib/ui/ui.v` warning: the following imports were never used: 
 * windows
```

**Comment by progamateur** [comment]
Fix warnings (V 0.1.27 53ffee1)

**Comment by medvednikov** [comment]
Do you have access to the CI logs for macos?

I think you don't, that's why I printed the error it's returning on macos.

**Comment by medvednikov** [comment]
Right now this PR only works on Windows.

**Comment by medvednikov** [comment]
And it's because you are doing `import windows` in ui.v

You should move that import to `ui_windows.c.v`

File: windows/v.mod [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Module {
+        name: 'windows'
+		description: 'shcore.dll',
+        version x64: Windows Kits\10\Lib\10.0.18362.0\um\x64
+		version x86: Windows Kits\10\Lib\10.0.18362.0\um\x86
```
Comment (dumblob) [inline windows/v.mod:5]: What is the license for the binaries (`x64/shcore.lib` and `x86/shcore.lib`) you're bundling with this PR?

Comment (progamateur) [reply windows/v.mod]: See Microsoft docs in windows_shcore folder:
  (1):  Microsoft sdk_license.pdf
  (2):  Microsoft Windows SDK redist lists _ Microsoft Docs.pdf

As seen in REDIST.TXT (2)
.
.
Static LIB files
Subject to the license terms for the software, the .lib files under the following directories may be distributed unmodified when built as part of your program:
Program Files\Windows Kits\10\Lib<version>\um\x86
Program Files\Windows Kits\10\Lib<version>\um\x64
.
.
As shcore.dll  is part of the above directories, I think it is distributable, but there are a lot of distribution requirements see in (1) 2.a.i. and ii.

**Comment by progamateur** [comment]
-Delete windows folder and and add windows_shcore folder.
-Add set_process_dpi_awareness function to ui_windows.c.v file

Note: for windows os, it is possible to install windows kit (with minimum components) without installing visual code to get shcore.dll

**Comment by dumblob** [comment]
>Note: for windows os, it is possible to install windows kit (with minimum components) without installing visual code to get shcore.dll

That's a valuable information - shouldn't we add it to some readme?

**Comment by progamateur** [comment]
@dumblob,
If access to shcore.dll via windows.shcore folder (as proposed in the PR) is not acceptable due to microsoft distribution requirements, the solution will be to install  windows kits independently of V UI (like freetype dependency). In that case a readme/installation file will be useful.

Yesterday, I did a minimum windows kits installation (last version) and that takes 1 GB anyway.

**Pull request closed without merging**